src/c++/vdo/base/status-codes.c

```diff
@@ -52,7 +52,9 @@ const struct error_info vdo_status_list[] = {
 	{ "VDO_BAD_NONCE", "Bad nonce" },
 	{ "VDO_JOURNAL_OVERFLOW", "Journal sequence number overflow" },
 	{ "VDO_INVALID_ADMIN_STATE", "Invalid operation for current state" },
+#if defined(VDO_INTERNAL) || defined(INTERNAL)
 	{ "VDO_CANT_ADD_SYSFS_NODE", "Failed to add sysfs node" },
+#endif
 };
 
```

Comment:
This is the new rebase conflict I mentioned; I added an INTERNAL error code bock so this could just get added to that.